v2/googlecloud-to-elasticsearch/src/main/java/com/google/cloud/teleport/v2/elasticsearch/templates/PubSubToElasticsearch.java

```diff
@@ -160,7 +160,7 @@ public static PipelineResult run(PubSubToElasticsearchOptions options) {
         .apply(
             "WriteTransformFailuresToBigQuery",
             ErrorConverters.WritePubsubMessageErrors.newBuilder()
-                .setErrorRecordsTable(options.getErrorOutputTable())
+                .setErrorRecordsTable(options.getErrorOutputTable().trim())
```

Comment:
Any reason to specify trim() only on this option as extra spaces might be possible for any options. If there is no specific reason, can remove trim() to maintain uniformity with other options.